docs/GettingStartedDocs/buildandsign.md

```diff
@@ -145,6 +150,18 @@ OE_SET_ENCLAVE_SGX(
     1);   /* NumTCS */
 ```
 
+To create an enclave at 0-base, defining the variable SGX_ENCLAVE_ZERO_BASE will create enclave with base address 0x0. Optionally, SGX_ENCLAVE_START_ADDR can be set to preffered enclave image start address.
```

Comment:
It might not be clear to an user what the difference between base address and start address is.
This needs a bit of clarification.